decisions.md

```diff
@@ -0,0 +1,5 @@
+# Decisions Regarding `vagov-content` Repository
+
+|Date | Decision | Endorsed By|
+|-- | -- | -- |
+|11/20/19 | VFS teams can have write access to this repository, which allows them to commit changes to master with only 1 code review, which can be a peer. Teams are still expected to submit content to platform (VSP) for review before publishing. This decision simply means that we are not enforcing that content review process with a technical block in the form of required code review from a certain person (since we do not enforce the content review process with a technical block in any other repo or the CMS). Decision to be re-evaluated in 4-6 months, and can be reversed if we find that folks are not following the content review process in this repo. | Nick Sullivan, Rian Fowler, Megan Kelley, Peggy Gannon, Patrick Bateman |
```

Comment:
Feel free to paste this right in the README if that'll increase visibility. Also, a decision title or shorthand description might make it a little easier to catch the gist.

Reply:
Ah yep good catch will do, thanks!